src/DTF/Libraries/WindowsInstaller.Package/InstallPackage.cs

```diff
@@ -190,7 +190,7 @@ public void ExtractFiles()
     public void ExtractFiles(ICollection<string> fileKeys)
     {
         this.ProcessFilesByMediaDisk(fileKeys,
-            new ProcessFilesOnOneMediaDiskHandler(this.ExtractFilesOnOneMediaDisk));
+            new ProcessFilesOnOneMediaDiskHandler(this.ExtractFilesOnOneMediaDisk));        
```

Comment:
Bah!  VS has been doing this to me a lot lately (I'm sure it is something that I am doing wrong, just don't know WHAT)!